bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -251,9 +251,13 @@ public HTableDescriptor getTableDescriptor(TableName tableName)
 
     try {
       return tableAdapter.adapt(bigtableTableAdminClient.getTable(request));
+    } catch (StatusRuntimeException e) {
+      if (e.getStatus().getCode() == Status.NOT_FOUND.getCode()) {
```

Comment:
Status is a class, not an enum: https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/Status.java#L62